### DISCUSSION TITLE
Tkhtml implementation

### DISCUSSION THREAD

**Discussion by Rieleniel** [alejandroautalan/pygubu-designer#277] [Ideas]
Dear Alejandro,

As before, I am still a grateful fan of your work with pygubu. Recently, I wanted to create a help page for my application via Markdown > HTML convarsion, so I can display it in Tkinter. I was going off of this stackoverflow discussion, and the answer's links to similar discussions.

Is it possible for you to implement Tkhtml into pygubu, like how you integrated customtkinter? I would be delighted to see my help page come alive through your package.

**Comment by alejandroautalan** [comment]
Hello @Rieleniel thanks for your kind words.

Pygubu already has a plugin for tkwinterweb package, which brings two widgets HtmlFrame and HtmlLabel.
Just install tkinterweb and the widgets will be shown.

Let me know if this works for you. 
If your need another package or question, let me know.

Regards
Alejandro A.

**Reply by Rieleniel** [reply]
Dear @alejandroautalan 

I have started using Tkinterweb. It was just a silly mistake on my part - I thought I installed Tkinterweb on the environment I was running, but I forgot.

That being said, there is a very curious error in the interaction between a Notebook widget and a HtmlFrame on 64-bit Windows, which some of my users run. [It is a known issue, and they have designed a drop-in replacement](https://github.com/Andereoo/TkinterWeb/blob/main/tkinterweb/docs/FAQ.md) (see Tkinterweb Issue#19 for more details). I do use Notebook widgets intensively in my application, but I run Linux, so I've only encountered this problem during live-testing.

Is it possible to implement this in Pygubu? (In case you already did and I'm blatantly blind, please let me know so I can touch grass and realign my chakras).

Kind regards,
Rieleniel

**Reply by alejandroautalan** [reply]
Hello @Rieleniel
As I used simple HtmlLabels, I didn't know about the extra widgets in the **tkinterweb.utilities** module.
I will try to implement them in the pygubu tkinterweb plugin ASAP.

Regards.
Alejandro A.
